src/vmm/src/arch/x86_64/msr.rs

```diff
@@ -271,159 +270,109 @@ pub fn get_msrs_to_save(kvm_fd: &Kvm) -> Result<MsrList, MsrError> {
     Ok(msr_index_list)
 }
 
-// List of MSRs that should not be included in the dump of CPU configuration.
+// List of MSRs that cannot be dumped.
 //
-// KVM_GET_MSR_INDEX_LIST returns some MSR indices that KVM_GET_MSRS fails to get (e.g., PMU,
-// VMX, MCE and Hyper-V related MSRs).
-//
-// Firecracker disables PMU by default in CPUID normalization for leaf 0xa. Due to this, PMU-
-// related MSRs cannot be gotten via KVM_GET_MSRS. The dependency on CPUID leaf 0xa can be found
-// in the following link.
+// KVM_GET_MSR_INDEX_LIST returns some MSR indices that KVM_GET_MSRS fails to get depending on
+// configuration. For example, Firecracker disables PMU by default in CPUID normalization for CPUID
+// leaf 0xA. Due to this, some PMU-related MSRs cannot be retrieved via KVM_GET_MSRS. The dependency
+// on CPUID leaf 0xA can be found in the following link.
 // https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/x86/kvm/vmx/pmu_intel.c?h=v5.10.176#n325
 //
-// We don't test if firecarcker works with nested virtualization environment. To avoid undefined
-// behavior, we exclude these VMX-related MSRs. You can see that VMX-related MSRs depend on whether
-// nested virtualization is allowed in the following link.
-// https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/x86/kvm/vmx/vmx.c?h=v5.10.176#n1950
-//
-// In kernel 4.14, IA32_MCG_CTL MSR can be gotten only if IA32_MCG_CAP.CTL_P[8] = 1 for vcpu.
-// IA32_MCG_CAP can be set up via KVM_X86_SETUP_MCE, but firecracker does not support this. To
-// avoid KVM_GET_MSRS failure on kernel 4.14, MCE-related MSRs are removed from the list.
-// https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/x86/kvm/x86.c?h=v4.14.311#n2553
-//
-// As firecracker does not work with Hyper-V, it is safe to ignore Hyper-V related MSRs.
-//
-// IA32_TSC MSR is for time stamp counter and can change as time goes on. KVM_GET_MSRS can get this
-// MSR safely, but should not be included in the dumped CPU configuration because it is used to
-// check diff between CPU models and detect changes of CPU configuration caused by firecracker/KVM/
-// BIOS changes.
-//
-// The list of MSRs that is potentially returned by KVM_GET_MSR_INDEX_LIST can be found in the
-// following link (`msrs_to_save_all` + `num_emulated_msrs`):
+// The list of MSR indices returned by KVM_GET_MSR_INDEX_LIST can be found in the following link
+// (`msrs_to_save_all` + `num_emulated_msrs`).
 // https://elixir.bootlin.com/linux/v5.10.176/source/arch/x86/kvm/x86.c#L1211
 static UNDUMPABLE_MSR_RANGES: &[MsrRange] = &[
```

Comment:
Same point as https://github.com/firecracker-microvm/firecracker/pull/4516#discussion_r1533615114, where possible it is better to use `const` over `static`.